src/Yoti.Auth/ShareUrl/Extensions/Extension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+
+namespace Yoti.Auth.ShareUrl.Extensions
+{
+    /// <summary>
+    /// Type and content of a feature for an application. Implemented <see cref="BaseExtension"/>, and adds generic content on top
+    /// </summary>
+    /// <typeparam name="T">Type of the extension's content</typeparam>
+    public class Extension<T> : BaseExtension
+    {
+        [JsonProperty(PropertyName = "content")]
+        private readonly T _content;
+
+        public Extension(string type, T content) : base(type)
+        {
+            _content = content;
+        }
+
+        /// <summary>
+        /// Get the feature's details
+        /// </summary>
+        /// <returns>The payload of the operation</returns>
+        public T Content
```

Comment:
@bucky-boy does Java (e.g. [GetLatitude](https://github.com/getyoti/yoti-java-sdk/blob/f682f2ca8f07a94c0e6b4a3b68062d7be6a624a1/yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/extension/LocationConstraintContent.java#L36)) have a specific need to make these getters public? I was thinking of making them internal in .NET. I wouldn't have though clients would need to retrieve these values since they are the ones passing them in to begin with

Reply:
As discussed, we're leaving them as public